packages/website/src/docs/happy-path.md

```diff
@@ -348,6 +362,8 @@ So far, we've been rolling nodes by hand. It's fun, but it's not the only way to
 
 Kits are collections of ready-made node factory functions for all types of nodes. Typically, kits are organized by purpose.
 
+Kits are an easy way to add functionality into your board without writing it yourself :) Think of kits as purpose-built third-party libraries you'd add into your web application.
```

Comment:
I thought this was a very interesting breadboard feature; but please double-check, let's make sure my description is accurate .